sgd/sgd-aa-oas3.yml

```
@@ -0,0 +1,577 @@
+        services) provided by the authenticated RP and having the current user
+        as delegator
+      security:
+        - dpopToken: []
+          dpopProof: []
+      parameters:
+        - name: DPoP
+          in: header
+          required: true
+          description: DPoP proof
+          schema:
+            "$ref": "#/components/schemas/DPoP"
+      x-spid-operation: # See §2.8
+        consentRequired: "SP"
+        spidLevel: SPID2
+        offlineAccess: false
+        offlineAccessDurationMax: 0
+      tags:
+        - protected
+      responses:
+        "200":
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DelegationList"
+              example:
+                delegation_list:
+                  - id: 1A00001AA21
+                    https://attributes.eid.gov.it/fiscal_number: TINIT-RSSMR*********I
+                  - id: 1A00002AA21
+                    https://attributes.eid.gov.it/fiscal_number: TINIT-ABCXX*********Z
+                  - id: 1A00004AA21
+                    https://attributes.eid.gov.it/fiscal_number: TINIT-DEFYY*********W
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+          #The requested service (or class of services) does not exist
+        "429":
+          $ref: "#/components/responses/429"
+        "500":
+          $ref: "#/components/responses/500"
+
+  /delegation/list/{service}:
+    get:
+      description: fdfd
+      operationId: get-delgation-service
+      summary:
+        "Returns the list of the delegations having the current user as
+        delegator and related to the services (or class of services) provided by
+        the authenticated RP and identifed from the service component in the URL"
+      security:
+        - dpopToken: []
+          dpopProof: []
+      x-spid-operation: # See §2.8
+        consentRequired: "SP"
+        spidLevel: SPID2
+        offlineAccess: false
+        offlineAccessDurationMax: 0
+      tags:
+        - protected
+      parameters:
+        - in: path
+          name: service
+          required: true
+          schema:
+            type: string
+          description:
+            Identifier of the service (or class of services) as given in the
+            online OAS3 catalog of delegable services provided by the
+            authenticated RP. It must be a UUID v4 identifier.
+          example: 21905575-991a-43d5-bede-25f59ae93373
+        - name: DPoP
+          in: header
+          required: true
+          description: DPoP proof
+          schema:
+            "$ref": "#/components/schemas/DPoP"
+      responses:
+        "200":
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DelegationList"
```

```suggestion
                $ref: "#/components/schemas/DelegationList"
```
Usare il plurale nei data type anziché il tipo, perchè in alcuni linguaggi c'è Collection o Array invece di List